### PR TITLE
dev-db/pgcli: bump to version 2.2.0.

### PR DESCRIPTION
**Pull request by dargor**
Bumped `dev-python/pgspecial`, needed.

### CODE REVIEW

**Comment by juippis** [comment]
Oh no, all was well until:
```
>>> Test phase: dev-db/pgcli-2.2.0
 * python3_5: running distutils-r1_run_phase python_test
================================== test session starts ===================================
platform linux -- Python 3.5.9, pytest-4.6.7, py-1.8.0, pluggy-0.13.1 -- /usr/bin/python3.5
cachedir: .pytest_cache
rootdir: /tmp/portage/dev-db/pgcli-2.2.0/work/pgcli-2.2.0
collecting ... collected 281 items / 6 errors / 275 selected

========================================= ERRORS =========================================
__________________ ERROR collecting tests/test_completion_refresher.py ___________________
ImportError while importing test module '/tmp/portage/dev-db/pgcli-2.2.0/work/pgcli-2.2.0/tests/test_completion_refresher.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.5/site-packages/_pytest/python.py:507: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib64/python3.5/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
/usr/lib64/python3.5/site-packages/_pytest/assertion/rewrite.py:304: in load_module
    exec(co, mod.__dict__)
tests/test_completion_refresher.py:3: in <module>
    from mock import Mock, patch
E   ImportError: No module named 'mock'
__________________________ ERROR collecting tests/test_main.py ___________________________
ImportError while importing test module '/tmp/portage/dev-db/pgcli-2.2.0/work/pgcli-2.2.0/tests/test_main.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.5/site-packages/_pytest/python.py:507: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib64/python3.5/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
/usr/lib64/python3.5/site-packages/_pytest/assertion/rewrite.py:304: in load_module
    exec(co, mod.__dict__)
tests/test_main.py:5: in <module>
    import mock
E   ImportError: No module named 'mock'
________________________ ERROR collecting tests/test_pgexecute.py ________________________
ImportError while importing test module '/tmp/portage/dev-db/pgcli-2.2.0/work/pgcli-2.2.0/tests/test_pgexecute.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.5/site-packages/_pytest/python.py:507: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib64/python3.5/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
/usr/lib64/python3.5/site-packages/_pytest/assertion/rewrite.py:304: in load_module
    exec(co, mod.__dict__)
tests/test_pgexecute.py:8: in <module>
    from mock import patch, MagicMock
E   ImportError: No module named 'mock'
________________________ ERROR collecting tests/test_rowlimit.py _________________________
ImportError while importing test module '/tmp/portage/dev-db/pgcli-2.2.0/work/pgcli-2.2.0/tests/test_rowlimit.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.5/site-packages/_pytest/python.py:507: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib64/python3.5/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
/usr/lib64/python3.5/site-packages/_pytest/assertion/rewrite.py:304: in load_module
    exec(co, mod.__dict__)
tests/test_rowlimit.py:2: in <module>
    from mock import Mock
E   ImportError: No module named 'mock'
___________ ERROR collecting tests/test_smart_completion_multiple_schemata.py ____________
ImportError while importing test module '/tmp/portage/dev-db/pgcli-2.2.0/work/pgcli-2.2.0/tests/test_smart_completion_multiple_schemata.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.5/site-packages/_pytest/python.py:507: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib64/python3.5/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
/usr/lib64/python3.5/site-packages/_pytest/assertion/rewrite.py:304: in load_module
    exec(co, mod.__dict__)
tests/test_smart_completion_multiple_schemata.py:4: in <module>
    from metadata import (
tests/metadata.py:8: in <module>
    from mock import Mock
E   ImportError: No module named 'mock'
___________ ERROR collecting tests/test_smart_completion_public_schema_only.py ___________
ImportError while importing test module '/tmp/portage/dev-db/pgcli-2.2.0/work/pgcli-2.2.0/tests/test_smart_completion_public_schema_only.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.5/site-packages/_pytest/python.py:507: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib64/python3.5/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
/usr/lib64/python3.5/site-packages/_pytest/assertion/rewrite.py:304: in load_module
    exec(co, mod.__dict__)
tests/test_smart_completion_public_schema_only.py:3: in <module>
    from metadata import (
tests/metadata.py:8: in <module>
    from mock import Mock
E   ImportError: No module named 'mock'
!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 6 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!
================================ 6 error in 0.88 seconds =================================
 * ERROR: dev-db/pgcli-2.2.0::testworld failed (test phase):
```

**Comment by dargor** [comment]
It's not clear to me where and how should mock be added, like this ?
```bash
BDEPEND="
	test? (
		dev-python/mock[${PYTHON_USEDEP}]
	)
"
```

Anyway, even with mock added :
- some tests need a local postgres instance running
- one test is failing, but it also fails with upstream corresponding release tag (master is fixed, though)

**Comment by dargor** [comment]
I read some ebuilds in dev-python, let me know if this is better.

File: dev-db/pgcli/pgcli-2.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	>=dev-python/click-4.1[${PYTHON_USEDEP}]
+	>=dev-python/configobj-5.0.6[${PYTHON_USEDEP}]
+	>=dev-python/humanize-0.5.1[${PYTHON_USEDEP}]
+	>=dev-python/pgspecial-1.11.8[${PYTHON_USEDEP}]
+	>=dev-python/pygments-2.0[${PYTHON_USEDEP}]
+	>=dev-python/setproctitle-1.1.9[${PYTHON_USEDEP}]
+"
+
+BDEPEND="
+	${RDEPEND}
```
Comment (juippis) [inline dev-db/pgcli/pgcli-2.2.0.ebuild]: I moved this to DEPEND as I saw it builds something during src_compile, but truthfully I'm not 100 % sure whether it belongs to DEPEND or BDEPEND. I'm not a python person :I 

But blame me if you get some whining about it.

**Pull request closed without merging**